src/main/java/org/broadinstitute/hellbender/cmdline/ArgumentCollection.java

```diff
@@ -13,4 +13,12 @@
 public @interface ArgumentCollection {
     /** Text that appears for this group of options in text describing usage of the command line program. */
     String doc() default "";
+
+    // The arguments in this collection are dependent on (and only enabled with the presence of)
+    // another argument. This may be used alone or together with dependsOnValue.
```

Comment:
We could add it for Argument as well - I just didn't want to do it for this PR since I think in this case it made more sense to use ArgumentCollection. If I *only* did it for Argument, it would mean that there would be references to the string "--readFilter" or "RF" inside of all the individual read filter classes.